### PR TITLE
Migrate Data Loss Prevention system tests according to AIP-47

### PR DESCRIPTION
**Pull request by bhirsz**
From #22447 and #22430

### CODE REVIEW

**Comment by bhirsz** [comment]
I now see that DLP have only 10 from 30 possible operators covered with system tests. I will update this PR with system tests - reverting PR to draft in meantime.

**Comment by bhirsz** [comment]
Added missing system tests except for CloudDLPRedactImageOperator operator which will require more work (mostly because we need to host images externally). Interesting fact: found out thanks to system tests that 2 operators were broken (I also fixed them in the PR).

File: tests/system/providers/google/cloud/data_loss_prevention/example_dlp_inspect_template.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    CloudDLPCreateInspectTemplateOperator,
+    CloudDLPDeleteInspectTemplateOperator,
+    CloudDLPGetInspectTemplateOperator,
+    CloudDLPInspectContentOperator,
+    CloudDLPListInspectTemplatesOperator,
+    CloudDLPUpdateInspectTemplateOperator,
+)
+from airflow.utils.trigger_rule import TriggerRule
+
+DAG_ID = "dlp"
```
Comment (o-nikolas) [inline tests/system/providers/google/cloud/data_loss_prevention/example_dlp_inspect_template.py]: Super small nit: DAG_ID does not match the file name.

**Review by o-nikolas — APPROVED** [review]

**Comment by potiuk** [comment]
Needs to be rebased and conflicts resolved.

**Comment by bhirsz** [comment]
@potiuk Rebased & fixed

**Comment by potiuk** [comment]
Almost @bhirsz  ... almost :)

**Review by potiuk — APPROVED** [review]

**Pull request merged**